Assets/Mirror/Runtime/Transport/Telepathy/Client.cs

```diff
@@ -71,24 +71,26 @@ void ReceiveThreadFunction(string ip, int port)
             {
                 // this happens if (for example) the ip address is correct
                 // but there is no server running on that ip/port
-                Logger.Log("Client Recv: failed to connect to ip=" + ip + " port=" + port + " reason=" + exception);
+                Logger.LogError($"Client Recv: failed to connect to ip={ip} port={port} reason={exception}");
 
                 // add 'Disconnected' event to message queue so that the caller
                 // knows that the Connect failed. otherwise they will never know
                 receiveQueue.Enqueue(new Message(0, EventType.Disconnected, null));
             }
-            catch (ThreadInterruptedException)
+            catch (ThreadInterruptedException exception)
             {
                 // expected if Disconnect() aborts it
+                Logger.LogError($"Client ThreadInterruptedException: failed with reason={exception}");
```

Comment:
`ThreadAbortException` and `ThreadInterruptedException ` shoudn't be Errors as they are expected when stopping a thread